CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## 2023-12-04
+
+[prod]
+
+- [associated PR](https://github.com/saritasa-nest/saritasa-devops-helm-charts/pull/101)
+- Release terraform-pod chart with 1.6.4 terraform version `0.0.20`
+
+## 2023-10-27
```

Comment:
Pls combine changelogs from one date to one block, i.e.

```
## 2023-10-27

[prod]

- [associated PR](https://github.com/saritasa-nest/saritasa-devops-helm-charts/pull/99)
- Added GitHub mergeable bot for checking PR  version `0.2.0`

- [associated PR](https://github.com/saritasa-nest/saritasa-devops-helm-charts/pull/98)
- Changed tekton-apps version to 0.2.15
```

Reply:
done